bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -147,7 +147,7 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
   // Keys Management
   CONFIG_PARAM(keyExchangeOnStart, bool, false, "whether to perform initial key exchange");
   CONFIG_PARAM(keyViewFilePath, std::string, ".", "TODO");
-
+  CONFIG_PARAM(publishClientsKeysOnStart, bool, false, "whether to publish client keys");
```

Comment:
Is it a rollout flag which eventually will be always true, or can it be disabled in future?

Reply:
I don't want this feature to be enabled by default in Apollo since it has side effects i.e. start sn, and it makes other tests like VC flaky